buildSrc/src/main/groovy/stairway.library-conventions.gradle

```diff
@@ -6,27 +6,16 @@ plugins {
     id 'stairway.java-conventions'
 }
 
-group = 'bio.terra'
-sourceCompatibility = JavaVersion.VERSION_11
-
-repositories {
-    mavenCentral()
-    google()
-}
-
-// Modify the standard :test task to only include unit-tagged tests.
-test {
-    useJUnitPlatform {
-        includeTags 'unit'
-    }
-}
-
 java {
     // Builds Javadoc into the published package as part of the 'assemble' task.
     withJavadocJar()
+
     // Builds sources into the published package as part of the 'assemble' task.
     withSourcesJar()
 }
+tasks.withType(Javadoc) {
+    options.addStringOption('Xdoclint:none', '-quiet')
```

Comment:
What does that do?

Reply:
There are 100+ warnings during java doc generation for missing comments - it expects a comment for every function, parameter, etc. 

Xdoclint:none --> This will avoid lint errors (typo, format) on the comments (https://www.baeldung.com/java-doclint)
quiet --> Removes warnings from display

Comments are recommended no doubt but warnings for Setters, Getters, and minor utils that are self-explanatory 

Sample warnings - 

/Users/dexamundsen/github/databiosphere/stairway/stairway/src/main/java/bio/terra/stairway/FlightDebugInfo.java:98: warning: no comment
  public void setRestartEachStep(boolean restart) {
              ^
/Users/dexamundsen/github/databiosphere/stairway/stairway/src/main/java/bio/terra/stairway/FlightDebugInfo.java:130: warning: no comment
  public void setUndoStepFailures(Map<String, StepStatus> undoStepFailures) {
              ^
/Users/dexamundsen/github/databiosphere/stairway/stairway/src/main/java/bio/terra/stairway/FlightDebugInfo.java:58: warning: no comment
    public Builder doStepFailures(Map<String, StepStatus> doStepFailures) {
                   ^